docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -20,74 +20,73 @@ import * as FileSystem from 'expo-file-system';
 
--   **`FileSystem.cacheDirectory`**
+- **`FileSystem.cacheDirectory`**
 
-  `file://` URI pointing to the directory where temporary files used by this app will be stored. Files stored here may be automatically deleted by the system when low on storage. Example uses are for downloaded or generated files that the app just needs for one-time usage.
+`file://` URI pointing to the directory where temporary files used by this app will be stored. Files stored here may be automatically deleted by the system when low on storage. Example uses are for downloaded or generated files that the app just needs for one-time usage.
 
 So, for example, the URI to a file named `'myFile'` under `'myDirectory'` in the app's user documents directory would be `FileSystem.documentDirectory + 'myDirectory/myFile'`.
 
-Expo APIs that create files generally operate within these directories. This includes `Audio` recordings,  `Camera` photos, `ImagePicker` results, `SQLite` databases and `takeSnapShotAsync()` results. This allows their use with the `FileSystem` API.
+Expo APIs that create files generally operate within these directories. This includes `Audio` recordings, `Camera` photos, `ImagePicker` results, `SQLite` databases and `takeSnapShotAsync()` results. This allows their use with the `FileSystem` API.
 
 Some `FileSystem` functions are able to read from (but not write to) other locations. Currently `FileSystem.getInfoAsync()` and `FileSystem.copyAsync()` are able to read from URIs returned by [`CameraRoll.getPhotos()`](https://facebook.github.io/react-native/docs/cameraroll.html#getphotos) from React Native.
 
--   **`FileSystem.EncodingTypes`**
+- **`FileSystem.EncodingTypes`**
 
-  These constants can be used to define how data is read / written.
+These constants can be used to define how data is read / written.
 
-  - **FileSystem.EncodingTypes.UTF8** -- Standard readable format.
+- **FileSystem.EncodingTypes.UTF8** -- Standard readable format.
 
-  - **FileSystem.EncodingTypes.Base64** -- Binary, radix-64 representation.
+- **FileSystem.EncodingTypes.Base64** -- Binary, radix-64 representation.
 
 ### `FileSystem.getInfoAsync(fileUri, options)`
 
 Get metadata information about a file or directory.
 
 #### Arguments
 
--   **fileUri (_string_)** -- `file://` URI to the file or directory, or a URI returned by [`CameraRoll.getPhotos()`](https://facebook.github.io/react-native/docs/cameraroll.html#getphotos).
+- **fileUri (_string_)** -- `file://` URI to the file or directory, or a URI returned by [`CameraRoll.getPhotos()`](https://facebook.github.io/react-native/docs/cameraroll.html#getphotos).
 
--   **options (_object_)** -- A map of options:
+- **options (_object_)** -- A map of options:
 
-    -   **md5 (_boolean_)** -- Whether to return the MD5 hash of the file. `false` by default.
+  - **md5 (_boolean_)** -- Whether to return the MD5 hash of the file. `false` by default.
```

Comment:
Are you sure a 2-space indentation is sufficient in this case? I think I remember it being to small for Markdown processor to notice, but we may have changed the settings since.

Reply:
Checked it once again, and it looks ok when run in dev. I am pretty sure other pages uses 2-space. It is automatically set by my VSCode, and other pages stays intact.